app/furniture/marketplace/marketplace.rb

```diff
@@ -22,6 +22,9 @@ class Marketplace < Furniture
     setting :stripe_webhook_endpoint
     setting :stripe_webhook_endpoint_secret
 
+    # Square order notifications integration
+    setting :square_location_id
```

Comment:
Is `location` a term-of-art in Square land?

Reply:
Yes. Business _location_ is a key aspect of a merchant account. It's required for account activation. There's a good overview here: https://developer.squareup.com/docs/locations-api.